src/vs/platform/theme/common/colorRegistry.ts

```diff
@@ -465,12 +465,12 @@ export const quickInputListFocusBackground = registerColor('quickInputList.focus
  * Menu colors
  */
 export const menuBorder = registerColor('menu.border', { dark: null, light: null, hcDark: contrastBorder, hcLight: contrastBorder }, nls.localize('menuBorder', "Border color of menus."));
-export const menuForeground = registerColor('menu.foreground', { dark: selectForeground, light: foreground, hcDark: selectForeground, hcLight: selectForeground }, nls.localize('menuForeground', "Foreground color of menu items."));
-export const menuBackground = registerColor('menu.background', { dark: selectBackground, light: selectBackground, hcDark: selectBackground, hcLight: selectBackground }, nls.localize('menuBackground', "Background color of menu items."));
+export const menuForeground = registerColor('menu.foreground', { dark: '#ffffff', light: foreground, hcDark: selectForeground, hcLight: selectForeground }, nls.localize('menuForeground', "Foreground color of menu items."));
+export const menuBackground = registerColor('menu.background', { dark: '#cccccc', light: selectBackground, hcDark: selectBackground, hcLight: selectBackground }, nls.localize('menuBackground', "Background color of menu items."));
```

Comment:
inline the `menu.background` color as opposed to using a different token means themes will get broken, here's the GH dark theme since there's no previous color token to inherit from:

![image](https://user-images.githubusercontent.com/35271042/167767095-a3c48ba3-0507-4f81-b3ad-b0b4012b5d5b.png)